ovos_tts_plugin_server/__init__.py

```diff
@@ -41,13 +44,8 @@ def verify_ssl(self) -> bool:
         """Whether or not to verify SSL certificates when connecting to the server. Defaults to True."""
         return self.config.get("verify_ssl", True)
 
-    def get_tts(
-        self,
-        sentence,
-        wav_file,
-        lang: Optional[str] = None,
-        voice: Optional[str] = None,
-    ) -> Tuple[Any, None]:
+    def get_tts(self, sentence, wav_file, lang: Optional[str] = None,
+                voice: Optional[str] = None) -> Tuple[Any, None]:
```

Comment:
This will fail CI because the `black` formatter will revert it back to the way it was